### PR TITLE
[8] Add page Navigation to your pages

### PR DESCRIPTION
**Pull request by MayowaShoyinka**
## Task: `[8] Add page Navigation to your pages
- `Linked my catalog-page to my contact-page
picture1
picture2
picture3

`

### CODE REVIEW

**Comment by adaugochi** [comment]
You have conflict on this file. This might be because you did sync or update your branch. The following steps below should help you fix this problem

**Comment by MayowaShoyinka** [comment]
Corrections Made

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 1]
```diff
@@ -13,17 +13,27 @@
             <ul>
                 <li><a href="#">Home</a></li>
                 <li><a href="#">About</a></li>
+<<<<<<< HEAD
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:16]: Remove this line @MayowaShoyinka

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 2]
```diff
@@ -13,17 +13,27 @@
...
+                <li><a href="../html/index.html">Catlog</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="contact.html">Contact</a></li>
+=======
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:20]: Remove this line @MayowaShoyinka

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 3]
```diff
@@ -13,17 +13,27 @@
...
                 <li><a href="#">Catlog</a></li>
                 <li><a href="#">Blog</a></li>
                 <li><a href="#">Contact</a></li>
+>>>>>>> master
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:24]: Remove this line

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 4]
```diff
@@ -13,17 +13,27 @@
...
 
             </ul>
         </div>
     </nav>
 
     <hr>
 
+<<<<<<< HEAD
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:32]: Remove this line

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 5]
```diff
@@ -13,17 +13,27 @@
...
+        <h3>Hi, Thank you for getting in touch. Fill out the form below, and we will get back to you immediately.</h3>
+=======
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:34]: Remove this line

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 6]
```diff
@@ -13,17 +13,27 @@
...
         <h3>Hi, Thank you for getting in touch. Fill out the form below, and we will get back to you immediately</h3>
+>>>>>>> master
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:36]: Remove this line

**Pull request merged**